cli/package.json

```diff
@@ -60,6 +60,7 @@
     "semver": "^7.3.7",
     "tar": "^6.1.11",
     "tslib": "^2.4.0",
+    "xml-js": "1.6.11",
     "xml2js": "^0.4.23"
```

Comment:
Looks like there are 2 libraries now `xml-js` and `xml2js`. Possible to refactor down to 1?